nss-day-cohort-41/tabloidcli-the-gonzo-group
# Dh posts listing

# Description
Ok! Posts looks pretty good now except for Notes.

# Testing Instructions
Main Menu > List Notes 4
Check 
1) List Posts
2) Add Post
3) Edit Post
4)  Remove Post

# Checklist:

![image](https://user-images.githubusercontent.com/1449595/93119377-51b42600-f687-11ea-8ced-2d50afab262d.png)
